src/server/pachyderm_test.go

```diff
@@ -4678,7 +4696,7 @@ func TestPipelineWithStats(t *testing.T) {
 	dataRepo := tu.UniqueString("TestPipelineWithStats_data")
 	require.NoError(t, c.CreateRepo(dataRepo))
 
-	numFiles := 500
+	numFiles := 10
```

Comment:
Similar to above, do we need to lower this?

Reply:
We need to lower this because list datum still needs performance work. It makes a bunch of random file requests, which works fine when the full hashtree fits in memory, but not so much when you have to go to object storage. This was the reason I brought up batching file requests based on a range so that we could grab a subtree using the indexing and have all the metadata we need at a point in time.

Reply:
Yeah, commands datum related. How fast is `inspect-datum` these days?

Reply:
Inspect datum is fine, and is much quicker on first access because it does not download the full hashtree. The access pattern where a datum fails then is inspected is much quicker in 1.8 for any reasonably sized filesystem.